### ISSUE TITLE
bug: threejs editor's export/import functions don't support polygonOffset

### ISSUE THREAD

**Issue by sterlingcrispin** [mrdoob/three.js#6254]
here's a scene.json with two objects at the same place, one should be offset but when importing into the editor these values seem to be ignored, if you export this scene after importing it those polygonOffset attributes that were present are no longer there http://pastebin.com/igFGGYXs

**Comment by mrdoob** [comment]
Out of curiosity... what are you using `polygonOffset` for ?

**Comment by sterlingcrispin** [comment]
this sort of thing with offsetting meshes in multimaterial objects, 
http://jsbin.com/ceyamovase/1/edit?html,js,output

it seems way more efficient then walking through all of the vertices of a mesh and dilate them along their normal or scaling the object 
http://stemkoski.blogspot.fr/2013/07/shaders-in-threejs-glow-and-halo.html

**Comment by satori99** [comment]
I use it to draw lines on top of geometry without z-fighting.

A GridHelper and a plane will z-fight if they have the same position unless the plane uses polygon offset.

There are a few things missing from the MaterialLoader that I often work around. I was about to do a PR to add `alphaTest` support to the MaterialLoader too.

**Comment by mrdoob** [comment]
Is it supposed to look like this?

![screen shot 2015-03-27 at 6 00 10 pm](https://cloud.githubusercontent.com/assets/97088/6877822/32db2dcc-d4ab-11e4-9afc-0c35c9dea923.png)

**Comment by mrdoob** [comment]
> There are a few things missing from the MaterialLoader that I often work around. I was about to do a PR to add `alphaTest` support to the MaterialLoader too.

Please do! :)

**Comment by WestLangley** [comment]
@mrdoob Yes, `polygonOffset` is useful in preventing z-fighting when a wireframe is rendered over a mesh.

	var material = new THREE.MeshPhongMaterial( {
		color: 0xff0000, 
		polygonOffset: true,
		polygonOffsetFactor: 1,
		polygonOffsetUnits: 1
	} );

	mesh = new THREE.Mesh( geometry, material );
	scene.add( mesh );

	var wfh = new THREE.WireframeHelper( mesh );
	scene.add( wfh );

**Comment by sterlingcrispin** [comment]
@mrdoob  yes it should look like that, object outlining, I have another version with a better looking vertex shader 
![screen shot 2015-03-27 at 3 16 39 pm](https://cloud.githubusercontent.com/assets/2976761/6877999/4f6dd4ec-d494-11e4-85ba-0837654a2d15.png)
